pkg/terminal/command.go

```diff
@@ -2353,13 +2353,17 @@ func PrintStack(formatPath func(string) string, out io.Writer, stack []api.Stack
 	d := digits(len(stack) - 1)
 	fmtstr := "%s%" + strconv.Itoa(d) + "d  0x%016x in %s\n"
 	s := ind + strings.Repeat(" ", d+2+len(ind))
+	printed := 0
 
 	for i := range stack {
+		if !filter(stack[i]) {
+			continue
+		}
 		if stack[i].Err != "" {
 			fmt.Fprintf(out, "%serror: %s\n", s, stack[i].Err)
 			continue
 		}
-		fmt.Fprintf(out, fmtstr, ind, i, stack[i].PC, stack[i].Function.Name())
+		fmt.Fprintf(out, fmtstr, ind, printed, stack[i].PC, stack[i].Function.Name())
```

Comment:
Wouldn't it be better to leave the stack frame number unaltered? So that it's going to be obvious that something was elided.

Reply:
+1

Reply:
Yeah this makes sense, especially since they are still in the call stack frame at this time.